cloud-functions/http_standardize_names/test_http_standardize_names.py

```diff
@@ -0,0 +1,81 @@
+import json
+from main import http_standardize_names
+from unittest import mock
+import pytest
+
+
+def test_standardize_names_bad_header():
+    request = mock.Mock(headers={"Content-Type": "not-application/json"})
+
+    assert http_standardize_names(request) == {
+        "status": 400,
+        "summary": "Bad request",
+        "description": "Header must inclue: 'Content-Type:application/json'.",
+    }
+
+
+def test_standardize_names_bad_body():
+    request = mock.Mock(headers={"Content-Type": "application/json"})
+    request.get_json.return_value = ""
+    with pytest.raises(BaseException):
+        http_standardize_names(request=request)
+
+
+def test_standardize_names_good_request():
+    request = mock.Mock(headers={"Content-Type": "application/json"})
+    str_request_body = {
```

Comment:
What's the purpose of the `str` in `str_request_body`. I realize this is a little nit picky, but if the purpose is to indicate that the value is a string I don't believe that is accurate given that it is being set to a dictionary.